### DISCUSSION TITLE
Open core download fails.

### DISCUSSION THREAD

**Discussion by Johnsmac** [community/community#135951] [Programming Help]
### Select Topic Area

Question

### GitHub Feature Area

Issues

### Body

I am new to Apple and I have inherited a Mid 2012 Mac Pro Tower that I have already upped the OS from High Sierra TO Mojave  to Catalina! I had no problems doing it after changing the GPU to metal capable! I would like to upgrade to at least Monterey and have been trying to do so with Legacy Patcher, but with no success! The download continually fails with "'connection lost" part way through the download and I get this notification when I try to restart it 'OpenCore-Patcher.pkg” cannot be opened because it is from an unidentified developer." I even tried downloading it to a memory stick with the same results. I've tried the latest version of the patcher and several of the earlier ones. What am I failing to do?

**Comment by queenofcorgis** [comment]
Hi @Johnsmac ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Apple and their products, like their [community](https://discussions.apple.com/welcome). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

_Please note for future discussions: The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._

**Comment by Johnsmac** [comment]
Thanks. But I have been to Apple and got nowhere.  I think it will be easier to just scrap the machine and stick with what I know, which is Microsoft!
________________________________

**Reply by RocaelLuisSantiago** [reply]
Hola
